packages/core/src/app/common/error/SentryErrorLogger.ts

```diff
@@ -135,6 +135,12 @@ export default class SentryErrorLogger implements ErrorLogger {
     }
 
     private handleBeforeSend: (event: Event, hint?: EventHint) => Event | null = (event, hint) => {
+        if (
+            event.breadcrumbs?.filter((breadcrumb) => breadcrumb.data?.url.includes('convertcart'))
```

Comment:
Do you for see us having a list of works like `convertcart` that we need to ignore in future?

Reply:
For other issues, we might have to use other signatures to identify them.
Right now, I think we just aim to remove `convertcart`.